docs/manage/_content.rst

```diff
@@ -56,12 +56,14 @@ To import content into Kolibri, follow these steps.
     Select a source to import Kolibri content channels.
 
 
+.. _import_studio:
+
```

Comment:
@ivanistheone I squeezed in a `manage.html#import-with-token` shortcut in the meantime, until I finish the refactor from #45 :wink: